src/components/RealTimeKeyword/RealTimeKeyword.style.ts

```diff
@@ -0,0 +1,108 @@
+import { ListItem, Typo } from '@yourssu/design-system-react';
+import styled from 'styled-components';
+
+interface StyledContainerProps {
+  containerPadding: string;
+  containerWidth: string;
+  containerHeight: string;
+}
+export const StyledContainer = styled.div<StyledContainerProps>`
+  position: relative;
+  padding: ${(prop: StyledContainerProps) => prop.containerPadding};
+  width: ${(prop: StyledContainerProps) => prop.containerWidth};
+  height: ${(prop: StyledContainerProps) => prop.containerHeight};
```

Comment:
```suggestion
  padding: ${(props) => props.$containerPadding};
  width: ${(props) => props.$containerWidth};
  height: ${(props) => props.$containerHeight};
```

StyledContainerProps type을 제너릭 형태로 명시해줬는데, 반복해서 작성할 필요가 없어보이네용
다른 부분도 이렇게 변경 해주시면 좋을 것 같아요

Reply:
```
padding: ${({ $containerPadding }) => $containerPadding};
width: ${({ $containerWidth }) => $containerWidth};
height: ${({ $containerHeight }) => $containerHeight};
```
[fix: styled-component props $ 컨벤션으로 수정](https://github.com/yourssu/Soomsil-Web/pull/189/commits/b8ff074fe21be6da0f122135c869193ecfa7eba1) 에서 이런 식으로 고쳤습니당